sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -21,9 +21,9 @@
 
 /// \defgroup sycl_graph DPC++ Execution Graph
 ///
-/// DPC++, unlike OpenCL, provides a programming model in which user doesn't
+/// DPC++, unlike OpenCL, provides a programming model in which the user doesn't
```

Comment:
Is there anything specific to DPC++ in the first sentence compared to SYCL?

Otherwise, something like "SYCL, unlike OpenCL,"...